src/toolkit/utils/noteService.js

```diff
@@ -0,0 +1,183 @@
+import axios from "axios";
+
+export const getNoteService = async (authToken) => {
+  try {
+    const response = await axios.get("/api/notes", {
+      headers: { authorization: authToken },
```

Comment:
Suggestion: you can create a interceptor on axios to add the token to all the requests instead of doing it in each method